src/lib/parse-purge.js

```diff
@@ -0,0 +1,62 @@
+const path = require('path');
+const fs = require('./sync-fs');
+const { warn } = require('./log');
+
+const throwError = (err) => {
+  throw new Error(`Error loading purge: ${err}`);
+};
+
+module.exports = (projectDir) => {
+  const purgePath = path.join(projectDir, 'purge.js');
+  const purgingPath = path.join(projectDir, 'purging.js');
+
+  const purgeExists = fs.exists(purgePath);
+  const purgingExists = fs.exists(purgingPath);
+
+  if (purgeExists && purgingExists) {
+    throwError(`Purge is defined at both ${purgePath} and ${purgingPath}.  Only one of these files should exist.`);
```

Comment:
Instead of "only one should exist" maybe reinforce that purge is deprecated and say "Use the purging.js file because purge.js is deprecated?" or alike.